<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Services</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="portField">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>161</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QListWidget" name="availableField">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>161</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="disableButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>90</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Disable</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Available</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enableButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="version">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QListWidget" name="nameField">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>151</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="package_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QListWidget" name="disableField">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>161</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Disabled</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
